accepted/future-releases/class-modifiers/feature-specification.md

```diff
@@ -551,9 +549,6 @@ The remaining valid combinations and their capabilities are:
 |`abstract base mixin class`|No     |**Yes**|No     |**Yes**|No     |
```

Comment:
This is not a new issue for this pull request, but I wanted to comment anyway:

Is it correct that `sealed class` has "Construct?" as "No" in this table?